### PR TITLE
Quickstart for ProxySQL

### PR DESCRIPTION
**Pull request by k1nect**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [✔️] Did you check you NRQL syntax? - Does it work?
- [✔️] Did you include an InstallPlan and Documentation reference?
- [✔️] Are all documentation links publically accessible?
- [✔️] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [✔️] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [✔️] Does the PR contain a screenshot for each of your dashboards?
- [✔️] Do your screenshots show data?
- [✔️] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [✔️] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by mickeyryan42** [comment]
Hey @k1nect I approved, but see a failed validation for quickstart schema.

File: alert-policies/ext-proxysql/active-transactions.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-proxysql/active-transactions.yml:1]:
```suggestion
```

File: alert-policies/ext-proxysql/alert.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-proxysql/alert.yml:1]:
```suggestion
```

File: alert-policies/ext-proxysql/slow-queries.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-proxysql/slow-queries.yml:1]:
```suggestion
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @k1nect, we actually don't want the `---` document separators in the alert conditions. The template versions were incorrectly formatted, sorry about that!

File: alert-policies/ext-proxysql/alert.yml (hunk 2 above)
Comment (bpschmitt) [reply alert-policies/ext-proxysql/alert.yml:1]: @k1nect this needs to be removed.  Looks like a boilerplate example.

**Pull request closed without merging**